brjs-sdk/sdk/libs/javascript/br-locale/src/switcher.js

```diff
@@ -32,6 +32,19 @@ module.exports = {
 		var normalizedPath = path === "/" ? "" : path.replace(/^\/?(.*?)(\/|\/index\.html)?$/, '/$1');
 		var localizedPath = normalizedPath + '/' + locale + ((fullyQualifiedPath) ? '.html' : '');
 
+		if (navigator.appName == 'Microsoft Internet Explorer') {
+			if (pageUrl.indexOf(":80") > -1) {
+				localizedPath = localizedPath.substring(1, localizedPath.length);
+			}
+			else if (host.indexOf(":80") > -1){
```

Comment:
Not really, it's for the case where the navigator is not IE, but the host port is explicitly asked for.